src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -864,10 +860,6 @@
           type: string
           short-summary: How outbound traffic will be configured for a cluster.
           long-summary: This option will change the way how the outbound connections are managed in the AKS cluster. Available options are loadbalancer, managedNATGateway, userAssignedNATGateway, userDefinedRouting, none and block. For custom vnet, loadbalancer, userAssignedNATGateway and userDefinedRouting are supported. For aks managed vnet, loadbalancer, managedNATGateway and userDefinedRouting are supported.
-        - name: --enable-pod-security-policy
-          type: bool
-          short-summary: Enable pod security policy.
-          long-summary: --enable-pod-security-policy is deprecated. See https://aka.ms/aks/psp for details.
         - name: --disable-pod-security-policy
```

Comment:
I didn't remove `--disable-pod-security-policy` because the user should be able to disable the feature. But after re-thinking, `--disable-pod-security-policy` is also to set the psp property to False. So `--disable-pod-security-policy` is also useless. It should be also removed.
 @FumingZhang  right?

Reply:
removed --disable-pod-security-policy